### PR TITLE
Update monero-core_zh-cn.ts

### PR DESCRIPTION
**Pull request by 4broaf**
ready for  review.

### CODE REVIEW

**Comment by erciccione** [comment]
ref #1377

**Comment by 4broaf** [comment]
ok i will finish it tonight.

File: translations/monero-core_zh-cn.ts [hunk 1]
```diff
@@ -743,12 +743,12 @@
     <message>
         <location filename="../components/ProgressBar.qml" line="38"/>
         <source>%1 blocks remaining: </source>
-        <translation type="unfinished"></translation>
+        <translation>区块剩余: %1</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: This should be `%1 区块剩余: `

File: translations/monero-core_zh-cn.ts [hunk 2]
```diff
@@ -743,12 +743,12 @@
...
     </message>
     <message>
         <location filename="../components/ProgressBar.qml" line="70"/>
         <source>Synchronizing %1</source>
-        <translation type="unfinished"></translation>
+        <translation>同步区块中: 1%</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: please remove `:`

File: translations/monero-core_zh-cn.ts [hunk 3]
```diff
@@ -784,128 +784,128 @@
...
         <location filename="../pages/Receive.qml" line="142"/>
         <source>Total received</source>
-        <translation type="unfinished"></translation>
+        <translation>全部已接收</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="165"/>
         <source>Set the label of the selected address:</source>
-        <translation type="unfinished"></translation>
+        <translation>标记已选地址</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: missing `:`

File: translations/monero-core_zh-cn.ts [hunk 4]
```diff
@@ -784,128 +784,128 @@
...
         <location filename="../pages/Receive.qml" line="373"/>
         <source>Create new address</source>
-        <translation type="unfinished"></translation>
+        <translation>创建新地址</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="388"/>
         <source>Set the label of the new address:</source>
-        <translation type="unfinished"></translation>
+        <translation>标记新地址</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: missing `:`

File: translations/monero-core_zh-cn.ts [hunk 5]
```diff
@@ -1022,12 +1022,12 @@
...
         <location filename="../pages/Settings.qml" line="680"/>
         <source> &lt;a href=&apos;#&apos;&gt; (Click to change)&lt;/a&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation> &lt;a href=&apos;#&apos;&gt; (点击改变)&lt;/a&gt;</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="682"/>
         <source>Set a new restore height:</source>
-        <translation type="unfinished"></translation>
+        <translation>设置新的区块恢复高度</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: missing `:`

File: translations/monero-core_zh-cn.ts [hunk 6]
```diff
@@ -1221,163 +1221,163 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/SharedRingDB.qml" line="189"/>
         <source>Load</source>
-        <translation type="unfinished"></translation>
+        <translation>载入</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="205"/>
         <source>Or manually blackball/unblackball a single output:</source>
-        <translation type="unfinished"></translation>
+        <translation>或: 手动选择单个balckball/unblackball输出</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: missing `:`

File: translations/monero-core_zh-cn.ts [hunk 7]
```diff
@@ -1221,163 +1221,163 @@ The old wallet cache file will be renamed and can be restored later.
...
-        <translation type="unfinished"></translation>
+        <translation>为了避免Monero环签名提供的保护无效, 同一个输出不应该在不同的区块链上使用不同的环签名来支付.  虽然这通常不是一个问题, 但是当复制了 Monero 密钥的克隆钱包允许您支付到现有输出时, 它可以成为一个问题.  在这种情况下, 您需要确保这些现有输出在两条链上都使用相同的环签名. &lt;br&gt;这将由Monero和任何不想破坏您隐私权的密钥重用软件自动完成.  &lt;br&gt; 如果您同时在使用复制了 Monero 密钥的克隆钱包, 并且不包含这种保护, 您仍然可以通过在它之上新的交易来确保您的交易安全, 然后在此页面上手动添加环签名, 从而使您安全地使用Monero. &lt;br&gt;如果您没有使用无此安全功能的克隆钱包, 那么您不需要做任何事情, 因为所有都是自动的. </translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: Please add `<br>` at the end

File: translations/monero-core_zh-cn.ts [hunk 8]
```diff
@@ -1221,163 +1221,163 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/SharedRingDB.qml" line="392"/>
         <source>Relative</source>
-        <translation type="unfinished"></translation>
+        <translation>相关的</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="408"/>
         <source>Segregation height:</source>
-        <translation type="unfinished"></translation>
+        <translation>隔离高度</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: missing `:`

File: translations/monero-core_zh-cn.ts [hunk 9]
```diff
@@ -2510,65 +2502,65 @@ Fee: </source>
...
         <location filename="../main.qml" line="551"/>
         <source>Daemon is synchronized</source>
-        <translation type="unfinished"></translation>
+        <translation>后台进程已同步</translation>
     </message>
     <message>
         <location filename="../main.qml" line="621"/>
         <source>Address: </source>
-        <translation type="unfinished"></translation>
+        <translation>地址</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: missing `:` + whitespace

File: translations/monero-core_zh-cn.ts [hunk 10]
```diff
@@ -2510,65 +2502,65 @@ Fee: </source>
...
-        <translation type="unfinished"></translation>
+        <translation>警告: 非默认签名环大小, 可能会破坏隐私, 推荐默认大小7</translation>
     </message>
     <message>
         <location filename="../main.qml" line="629"/>
         <source>
 
 Number of transactions: </source>
-        <translation type="unfinished"></translation>
+        <translation>交易次数</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: missing `:` + whitespace

File: translations/monero-core_zh-cn.ts [hunk 11]
```diff
@@ -2510,65 +2502,65 @@ Fee: </source>
...
 Description: </source>
-        <translation type="unfinished"></translation>
+        <translation>描述: </translation>
     </message>
     <message>
         <location filename="../main.qml" line="632"/>
         <source>
 Spending address index: </source>
-        <translation type="unfinished"></translation>
+        <translation>支付(Spending)地址目录</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: missing `:` + whitespace

File: translations/monero-core_zh-cn.ts (hunk 1 above)
Comment (4broaf) [reply translations/monero-core_zh-cn.ts]: ok, thanks for your review! :)

File: translations/monero-core_zh-cn.ts (hunk 11 above)
Comment (4broaf) [reply translations/monero-core_zh-cn.ts]: have done. sorry for so many errors.

**Comment by erciccione** [comment]
@4broaf Can you squash those commits in just one? if you don't know how to do it just add me as a collaborator to this repo, i will do it for you

**Comment by 4broaf** [comment]
@erciccione 
Sorry I'm not familiar with the use of github, have added you as collaborator. 
Thnaks.

**Comment by erciccione** [comment]
@luigi1111 ^ good to go

**Pull request merged**